pyxem/generators/indexation_generator.py

```diff
@@ -122,6 +123,54 @@ def get_library_FT_dict(template_library, shape, fsize):
     return library_FT_dict
 
 
+class RotationIndexationGenerator:
+    """Generates a template-based indexer that also calculates relative
+    rotation of templates.
+
+    Parameters
+    ----------
+    signal : ElectronDiffraction2D
+        The signal of electron diffraction patterns to be indexed.
+    diffraction_library : DiffractionLibrary
+        The library of simulated diffraction patterns for indexation.
+
+    Notes
+    -----
+    To be used with minimal template libraries whereby the first euler
+    angle is 0. It is this angle that is optimized during indexation.
+    """
+    def __init__(self, signal, diffraction_library):
+        self.signal = signal
+        self.library = diffraction_library
+
+    def correlate(self,
+                  n_largest=5,
+                  method="fast_correlation",
```

Comment:
I think we can drop method here unless you've got plans to build alternative metrics in the very near future.